src/arb/arb.jl

```diff
@@ -2158,3 +2158,47 @@ end
 ################################################################################
 
 # see inner constructor for ArbField
+
+################################################################################
+#
+#  Random generation
+#
+################################################################################
+
+@doc Markdown.doc"""
+    rand(r::ArbField; randtype::Symbol=:null)
```

Comment:
Maybe it's best to keep the `rand(r::ArbField)` API for generating uniformly a random value in `[0,1]` like is usually expected for floats generation, to the extent it makes sense for `ArbField`. A default implementation might be `r(rand(BigFloat))` for example. So I would either not give a default value for `randtype` here, or name the function `randtest`.

Reply:
I think this should be resolved now. Thank you for your review.